org.scummvm.scummvm.json

```diff
@@ -0,0 +1,151 @@
+{
+    "app-id": "org.scummvm.scummvm",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "24.08",
+    "sdk": "org.freedesktop.Sdk",
+    "finish-args": [
+        "--device=all",
```

Comment:
Required for joysticks and other devices. Will probably be replaced with the USB portal in the future, but we are not there yet.

Reply:
>  Will probably be replaced with the USB portal in the future, but we are not there yet.

You misunderstand what the USB portal is for.

Reply:
How?